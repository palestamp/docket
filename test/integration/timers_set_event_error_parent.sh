rm ~/.timer
set -o verbose
docket timer new work --abstract
docket timer new work_child --parent work
docket timer new work_child2 --parent work
docket timer new work_child3 --parent work

docket timer set work_child event:onstart:stop work
