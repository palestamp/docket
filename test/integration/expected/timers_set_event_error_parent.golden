docket timer new work --abstract
docket timer new work_child --parent work
docket timer new work_child2 --parent work
docket timer new work_child3 --parent work

docket timer set work_child event:onstart:stop work
error: Event can not affect parents (work_child parent of work)
